### DISCUSSION TITLE
What's the password when start the standalone_embed.sh script?

### DISCUSSION THREAD

**Discussion by lingvisa** [milvus-io/milvus#35475] [Q&A and General discussion] [answered]
Hello! I am trying to start the container script by following the instruction here:
`https://milvus.io/docs/install_standalone-docker.md`

```
bash standalone_embed.sh start
Password:
```

What's the password? This is my first time to install it by following the instruction. I am on MacBookPro 14.5.

**Comment by lingvisa** [comment, marked as answer]
It should be my own admin password and i got it solved. Thank you.

**Reply by aqiao** [reply]
Same question, what's the admin password you mentioned?

**Reply by yhmo** [reply]
@aqiao 
Open the **standalone_embed.sh** by a text editor, you will see this script uses "sudo" to launch docker container. So, it asks the password of your system account.
![Screenshot from 2025-04-01 18-11-41](https://github.com/user-attachments/assets/f96f4874-b947-41df-90de-28ae722b3366)

**Comment by aqiao** [comment]
Got it ! Thanks @yhmo
